### ISSUE TITLE
First Issue about Training

### ISSUE THREAD

**Issue by sainttelant** [weningerleon/TextileDefectDetection#2]
if I activate the b_train_cnn, the codes will run into the training process, however, dirs_groundtruth is an empty list, how did you design it to go??? 
or could you plz tell me the what is dirs_groundtruth? what does it look like?  how to set it up as a global parameters,  thanks a lot, i will appreciate you so much.

**Comment by sainttelant** [comment]
I  guess it should be a folder Path, however , what is the format of training dataset?

**Comment by weningerleon** [comment]
Hi, 
Exactly, its a folder path, where groundtruth labels of the images you use for training can be found.
In these images, every pixel needs to be label as weft, warp or in-between. You can have a look at the output of the FCN in example/FCN_result/res01.png. Training images should look similar to this

**Comment by sainttelant** [comment]
@weningerleon thanks for your kindly replying, however, when i checked the reso1.png as you mentioned , i am still confused about your datasets, could you plz update one or two pic and label groups as examples?  did you label them via using labelImg or some relevant labeling tools?
